piet-scene/src/scene/builder.rs

```diff
@@ -273,6 +280,16 @@ impl<'a> SceneBuilder<'a> {
     }
 }
 
+/// Returns false if a brush would be guaranteed to render nothing.
+fn check_brush(brush: &BrushRef) -> bool {
```

Comment:
I'd suggest `is_empty` or `renders_nothing` (together with inverting the return value) since that is less ambiguous than `check_brush`.

Reply:
Makes sense— I’ll give it a better name. Thanks!